src/color.rs

```diff
@@ -78,6 +78,7 @@ impl ColorType {
 ///
 /// Another purpose is to advise users of a rough estimate of the accuracy and effort of the
 /// decoding from and encoding to such an image format.
+#[allow(missing_docs)]
```

Comment:
I'd like to add doc comments here like `ColorType` rather than suppress warnings.

Reply:
Done